packages/context/src/__tests__/acceptance/bind-decorator.acceptance.ts

```diff
@@ -72,4 +78,20 @@ describe('@bind - customize classes with binding attributes', () => {
       'controllers.my-controller',
     ]);
   });
+
+  it('honors binding scope from @bind', () => {
+    let binding = createBindingFromClass(MySingletonController, {
+      defaultScope: BindingScope.TRANSIENT,
```

Comment:
nitpick: it would be good to mention the precedence in Binding's doc: https://loopback.io/doc/en/lb4/Binding.html#configure-binding-attributes-for-a-class

Reply:
Added.